keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/bakery.kt

```diff
@@ -1,5 +0,0 @@
-package com.netflix.spinnaker.keel.bakery
```

Comment:
RIP 👋

Reply:
Yeah, nothing was using it already. I just happened to notice.